apps/crn-frontend/src/network/working-groups/__mocks__/api.ts

```diff
@@ -18,3 +20,10 @@ export const getWorkingGroups = jest.fn(
   async (pageSize): Promise<WorkingGroupListResponse> =>
     createWorkingGroupListResponse(pageSize ?? 10),
 );
+
+export const createWorkingGroupResearchOutput: jest.Mocked<
+  typeof originalCreateWorkingGroupResearchOutput
+> = jest.fn(async () => ({
+  ...createResearchOutputResponse(),
+  id: 'research-output-id',
```

Comment:
Surely this at the very least should override the ownership. Won't this return data that isn't applicable to working group outputs?